app/src/main/java/ch/epfl/sdp/blindly/main_screen/MapFragment.kt

```diff
@@ -0,0 +1,49 @@
+package ch.epfl.sdp.blindly.main_screen
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+import ch.epfl.sdp.blindly.R
+import ch.epfl.sdp.blindly.UserMapActivity
+import com.google.android.gms.maps.model.LatLng
+
+
+class MapFragment : Fragment() {
+
+    companion object {
+        private const val ARG_COUNT = "messageArgs"
```

Comment:
You should change the string inside the ARG_COUNT to "mapArgs" ;)

Reply:
Indeed :)